RadarSDK/RadarReplayBuffer.m

```diff
@@ -42,6 +43,23 @@ - (void)writeNewReplayToBuffer:(NSMutableDictionary *)replayParams {
     // add new replay to buffer
     RadarReplay *radarReplay = [[RadarReplay alloc] initWithParams:replayParams];
     [mutableReplayBuffer addObject:radarReplay];
+
+    NSMutableArray<RadarReplay *> *prunedBuffer;
+
+    // If buffer length is above 50, remove every fifth replay from the persisted buffer 
+    if ([mutableReplayBuffer count] > 50) {
```

Comment:
hmm, wondering under what circumstances this would happen? when would `/track/replay` realistically return a non-200?

could be wrong but if I understand correctly, the endpoint returns a 201 and does everything async after the fanout - regardless, we should handle SQS being down gracefully

Reply:
In early replay testing, I witnessed some 413 Payload Too Large's